### ISSUE TITLE
Issues with Linux install README file

### ISSUE THREAD

**Issue by scaleoutsean** [solidfire/PowerShell#15]
I don't want to create a PR for this because there are different ways to fix these.

1) At the beginning, `sudo` is used, but later it isn't. This leads the user to think that sudoer is not required later, which isn't true. You could remove sudo altogether and mention that it should be used for actions outside of the user account's $HOME, or you could add sudo to other places where it's missing.

2) [Install commands](https://github.com/solidfire/PowerShell/blob/master/docs/linux/README.md#next-install-solidfire-bits) are hardcoded to alpha 14 which is not good because it creates issues when the code is updated (e.g. now `powershell_6.0.0-alpha.15-1ubuntu1.16.04.1_amd64.deb`). Rather than change that 14 to 15 and have it wrong as soon as a new release is out, I propose that you rework the doc to use a `$VERSION` variable. Or the README file could specify which PS version should be used and link to the exact release which you want people to use (right now it doesn't say if a specific minor version has to be used and links to the main PowerShell releases page which has a newer release).

**Comment by ahaid** [comment]
I acknowledge this issue and I'm coming up with a solution.

**Comment by scaleoutsean** [comment]
I just installed v6.00 RC1 on Ubuntu 17 using MS repos (the more recent approach) and it was seamless.  (I had to use slightly modified shortcuts, etc. to match this version.)
According to MS, repository based installs are the preferred method for all Ubuntu, Deb & CentOS versions, which could potentially be a source of simplification. Or new complications, if most people use the recommended approach and install PowerShell updates on auto-pilot, as we move through RC2, RC3, etc.
